### PR TITLE
Create free-programming-books-ml.md

### PR DESCRIPTION
**Pull request by arnavjune2**
Added free reading content for all the ml enthusiasts out there!

## What does this PR do?
Add resources

### Why is this valuable (or not)?
This is for all the machine learning enthusiast who are learning and implementing machine learning algorithms

### How do we know it's really free?
I have gone through the resources all by myself.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-ml.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+* [Machine Learning](#machine-learning)
+
+
+### Machine Learning
+
+* [Hands-On Machine Learning with Scikit-Learn and TensorFlow \| English](https://www.pdfdrive.com/hands-on-machine-learning-with-scikit-learn-and-tensorflow-concepts-tools-and-techniques-to-build-intelligent-systems-e168440497.html)) - Aurelien Geron
+* [Machine Learning For Absolute Beginners \| English](https://bmansoori.ir/book/Machine%20Learning%20For%20Absolute%20Beginners.pdf)) - Oliver Theobald (PDF)
+* [Machine Learning For dummies \| English](https://www.ibm.com/downloads/cas/GB8ZMQZ3) - Judith Hurwitz and Daniel Kirsch (PDF)
+* [Pattern Recognition and Machine Learning\| English](https://www.microsoft.com/en-us/research/uploads/prod/2006/01/Bishop-Pattern-Recognition-and-Machine-Learning-2006.pdf) - M. Jordan J. Kleinberg (PDF)
```
Comment (davorpa) [inline books/free-programming-books-ml.md:11]:
```suggestion
* [Pattern Recognition and Machine Learning\| English](https://www.microsoft.com/en-us/research/uploads/prod/2006/01/Bishop-Pattern-Recognition-and-Machine-Learning-2006.pdf) - Christopher M. Bishop (PDF)
```

File: books/free-programming-books-ml.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+### Index
+
+* [Machine Learning](#machine-learning)
+
+
+### Machine Learning
+
+* [Hands-On Machine Learning with Scikit-Learn and TensorFlow \| English](https://www.pdfdrive.com/hands-on-machine-learning-with-scikit-learn-and-tensorflow-concepts-tools-and-techniques-to-build-intelligent-systems-e168440497.html)) - Aurelien Geron
+* [Machine Learning For Absolute Beginners \| English](https://bmansoori.ir/book/Machine%20Learning%20For%20Absolute%20Beginners.pdf)) - Oliver Theobald (PDF)
```
Comment (davorpa) [inline books/free-programming-books-ml.md:9]: Copyrighted content. Please remove

File: books/free-programming-books-ml.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+* [Machine Learning For dummies \| English](https://www.ibm.com/downloads/cas/GB8ZMQZ3) - Judith Hurwitz, Daniel Kirsch (PDF)
+* [Pattern Recognition and Machine Learning\| English](https://www.microsoft.com/en-us/research/uploads/prod/2006/01/Bishop-Pattern-Recognition-and-Machine-Learning-2006.pdf) - Christopher M. Bishop (PDF)
```
Comment (davorpa) [inline books/free-programming-books-ml.md:11]: Not a malayalam resources. could you move to english books listing?

**Review by davorpa — CHANGES_REQUESTED** [review]

**Comment by eshellman** [comment]
ml  = malayalam  not machine learnign

**Pull request closed without merging**